### PR TITLE
Alex Caves

### PR DESCRIPTION
**Pull request by AlexanderCaves**

### CODE REVIEW

**Comment by Mister-Corn** [comment]
Don't forget to write a description about what you are doing today and tomorrow!

**Comment by AlexanderCaves** [comment]
Wednesday:
install dependencies
organize tables
migrations
seeds
authentication
endpoints
web tokens

Thursday:
create react app
organize routes/components
create components
create routes
signout button

File: db/seeds/users.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        {user_name: 'Yasiel Puig', password: '', department: 'Outfield'},
+        {user_name: 'Kike Hernandez', password: '', department: 'Utility'},
+        {user_name: 'Chris Taylor', password: '', department: 'Utility'},
+        {user_name: 'Kenta Maeda', password: '', department: 'Rotation'},
+        {user_name: 'Joc Pedersen', password: '', department: 'Outfield'},
+        {user_name: 'Cody Bellinger', password: '', department: 'Infield'},
+        {user_name: 'Scott Alexander', password: '', department: 'Bullpen'},
+        {user_name: 'Manny Machado', password: '', department: 'Infield'},
+        {user_name: 'Yasmani Grandal', password: '', department: 'Infield'},
+        {user_name: 'Ross Stripling', password: '', department: 'Rotation'},
```
Comment (Mister-Corn) [inline db/seeds/users.js]: If you're making a seed file, it's probably a good idea to import `bcrypt` in this file and hash password before they are inserted into the server. DM me if you want more information about this.

**Comment by Mister-Corn** [comment]
Alex, you were supposed to use JWT's in this lab... 😅

**Pull request closed without merging**